### ISSUE TITLE
14 Days wallet no syn'd. Coin somewhere out there...help please

### ISSUE THREAD

**Issue by RyanS34** [amjuarez/bytecoin#103]
I bought a significant amount of BCN 2 weeks ago...sent to wallet version 1.1.1 via Changelly for storage...but wallet is still not syn'd and coins are out there somewhere.  A significant purchase was made.  I need some help with this please.

I am a notice to the technology, so with help please keep it simple..and thanks.

**Comment by nibbons** [comment]
Have no fear.  Does your wallet try to sync and going slow, or is it stalled?

You can always delete your blockchain files in "C:\Users\yourpcname\AppData\Roaming" and sync from scratch. Delete these files (SAVE YOUR WALLET<---):

blockchainindices.dat
blockindexes.dat
blocks.dat
blockscache.dat

Verify that you have a ".wallet" file in this folder. DO NOT delete this...that is your wallet just make sure you keep that, maybe back it up right now into another folder. Now that the blockchain is deleted re-open the BCN wallet and it should sync. Let it go overnight or something, it's actually not that big but make sure the window is up and just let it go.

**Comment by RyanS34** [comment]
I bought a grip of BCN before this recent pump...755K coins, in fact.  I installed the software for the wallet..opened and the wallet just in in permanent sync....that is all that happens; nothing else...it did at one time say it was fully sync'd but zero BCN were in wallet..the next time I opened the wallet it was syncing again.  Again, I am a newb at this; so please bear with me and my questions.  

So, delete all 4 of those files...and then reopen wallet?...yes, I read do not delete .wallet.....is that correct?

**Comment by Rocherd** [comment]
YES keep the .wallet had a similar issue with this in Linux. The fix for it was deleting the .bytecoin folder. Im not 100% 
sure how it translates into Windows (my guess is somewhere in appdata.), but the Linux version completely rebuilds a new .bytecoin folder. Let sync, replace new wallet, with yours. Someone who has more experience with Windows, please clean up after me, as I am doing too much guessing.

**Comment by RyanS34** [comment]
i do appreciate all the help...I was thinking that BCN was scam and was filing a fraud complaint with my credit card...will look into these in the morning.....any other details from any else would be greatly welcomed.

**Comment by Rocherd** [comment]
What's your os?

**Comment by RyanS34** [comment]
windows 8

**Comment by Rocherd** [comment]
Could be your uac settings, someone confirm, please. If you do change them change only in small ways.

**Comment by RyanS34** [comment]
ok..so i was able to delete 2 of the files listed as those were the only ones there.  The wallet never reopened. I got an error box that stated the wallet was already running...but it was not.  So, what is the next step?  This is incredibly frustrating...I am losing ALOT of money because I cannot access these coins.

**Comment by nibbons** [comment]
It sounds like the wallet wasn't terminated all the way before deleting files.  Just cntrl+alt+delete if you are still having the issue and end the task, and then delete the files.  It works fine and I am just a fan here to help, the rest is up to you.  Maybe someone else on the subreddit can help you: reddit.com/r/bytecoinbcn

**Comment by RyanS34** [comment]
delete ALL but .wallet?

**Comment by nibbons** [comment]
delete these files:

blockchainindices.dat
blockindexes.dat
blocks.dat
blockscache.dat

**Comment by RyanS34** [comment]
i do not have all of those on the C drive..this is what I have located there:
DB
blockindexes
blocks
bytecoinwallet.cfg
bytecoinwallet.lock
bytecoinwallet.wallet
core
p2pstate
wallet
walletgui

I deleted the blockindexes and block the first time...and go the error message as previous stated.

Surely, users do not have to go through this process each time they try to sync their wallets?
So, ,what now?
Also..Nibbons, are you on the BCN DevTeam?

**Comment by RyanS34** [comment]
![screenshot 5](https://cloud.githubusercontent.com/assets/28837559/26365805/b675284a-3fb6-11e7-9bb9-476d77e6eeca.png)

**Comment by nibbons** [comment]
My bad ok delete: blocks, blockindexes and the whole DB folder

**Comment by RyanS34** [comment]
ok..to confirm...delted those ONLY..there reopen teh BCN wallet...let it run for 8 hours or so....and I should be good to go?...will the coins that I had transferred to this wallet address automatically load once the wallet is syn'd?

**Comment by nibbons** [comment]
Yeah the coins that you transferred will automatically load once the wallet is synced.  I dont know how long it takes (it depends on a few things), but try it out and it should show you the progress now.

**Comment by nibbons** [comment]
PS Jaxx is adding support for BCN so eventually you wont have to download the whole blockchain

**Comment by jp1302** [comment]
Hello, I am having the same issue as @RyanS34, Downloaded the bytecoin official wallet from the official webpage: bytecoin.org. The wallet lasted almost 7 days on the synced it, now that it says it is done with the syncing phase it dosent show me the 412k bytecoins I bought from changelly last week. This is a huge problem since I bought them when they were cheap, and now I have nothing. Can someone help me out with this problem since I have sent like 4 emails to the contact email bytecoin has on their webpage and no one has answered it.

**Comment by nibbons** [comment]
@jp1302 can you confirm that your wallet has the correct address that you used at changelly?  and what does your wallet status bar say? (it should look something like this but with most current time and date: http://imgur.com/a/3aSLv)

**Comment by nibbons** [comment]
@RyanS34 Im not a dev I just sub'd to the issues area because I saw there were a lot of unanswered questions I could help with, so I get an email anybody says anything :p

**Comment by jp1302** [comment]
@nibbons yep my wallet is the same address I put on changelly. I have the receipt and everything. 

Yes the status bar says this: Wallet syncchronized. Top block height: 1216321 / time (UTC): 07 Mar 2017, 23:41:52 Warning: last block was received 1841 hours 39 minutes ago.    8 Peers

**Comment by RyanS34** [comment]
ok..thanks for the help....if this gets up and running I wanna pay you some BCN as my appreciation.
But his is what remains after I deleted those files...about to reopen wallet and see what happens...will let ya now tomorrow some time..thanks again.

**Comment by RyanS34** [comment]
![screenshot 6](https://cloud.githubusercontent.com/assets/28837559/26367085/71ce57fc-3fbb-11e7-8d08-b981350c9e53.png)

**Comment by jp1302** [comment]
I cant find the file with that info. I g
![byte](https://cloud.githubusercontent.com/assets/28902985/26367472/515dbbb8-3fb4-11e7-9359-29bc7604eccd.png)
ot this

**Comment by jp1302** [comment]
Thats the only file I can find with information of my wallet. @nibbons

**Comment by nibbons** [comment]
@jp1302 browse to "C:\Users\yourpcname\AppData\Roaming\bytecoin" and that's where the files are.  Make sure you only delete blocks.bin, blockindexes.bin and the DB folder (don't delete your wallet).

@RyanS34 no problem I hope it works out for you this time

**Comment by jp1302** [comment]
@nibbons just did that, and when I open the program again it appears the same files I just deleted. Now the program is starting to sync again, hope the problem is fixed now and my BCN are there!

**Comment by nibbons** [comment]
@jp1302 great to hear!  That is exactly what it should do so you seem to be on your way.  I hope it syncs sooner rather than later, gl!

**Comment by nibbons** [comment]
@jp1302 Dev released update today with sync fix if you still have issues: https://twitter.com/bytecoin_dev/status/867122010668793859

**Comment by Patosky13** [comment]
Im having same issue. My wallet Syn since 3 days now. The problem is that i have sent significant amount of coins to this wallet through Changelly . How do i handle this.

**Comment by Ben19904** [comment]
@nibbons 

Does that new release help with the problem of not receiving bytecoin (despite being fully synced with the network?)

If so, how do I safely download this and recover my bytecoin?

**Comment by RyanS34** [comment]
progress so far after 24 hours...and as BCN drops, I am losing in the thousands of dollars.
![screenshot 10](https://cloud.githubusercontent.com/assets/28837559/26428676/fc233a62-40b0-11e7-9c22-51c7d9cbe065.png)

**Comment by RyanS34** [comment]
tried new version 1.1.3 aaaannndddd:
![screenshot 11](https://cloud.githubusercontent.com/assets/28837559/26428964/55506c30-40b2-11e7-9b31-c4cdc1cb01b2.png)

**Comment by boomworking** [comment]
let's use online light wallet freewallet.org
if you not android just use Andy to simulate for temporary usage.
until they release iOS version﻿.

**Comment by Skagnasty** [comment]
I had the same issue with 1.1.4 running on Win 8.1 Pro, 64-bit.  It took three attempts to resynch, but finally it did fully resynch and all is well.  Not sure why it took 3 times to delete the above mentioned files/directory, and then wait for it to resynch eac h time.  Once it does work, the app looks good.

**Comment by jplakali** [comment]
i transferred some bytecoin from cryptonator exchange to bytecoin desktop wallet and trying to sync the wallet but its syncing at very slow speed. i changed the node to 8080 and my wallet synced in 5 minutes but my wallet balance was still 0 so i changed the node to automatic and it started syncing again from 300k blocks. is there any way to take BCN out of the wallet without syncing the wallet? i can see confirmed transaction on chainradar. please help me.
